elements/rh-autocomplete/src/rh-search-droplist.scss

```diff
@@ -0,0 +1,63 @@
+@import "../rh-sass/rh-sass";
+
+:host {
+  position: relative;
+  display: none;
+  font-family: var(--rh-theme--font-family);
+  font-size: var(--rh-theme--font-size);
+  line-height: var(--rh-theme--line-height);
+}
+
+:host([open]) {
+  display: block;
+}
+
+.sr-only {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
+  border: 0;
+}
+
+.droplist {
+  display: block;
+  position: absolute;
+  top: 100%;
+  left: 0px;
+  right: 0px;
+  z-index: 9999;
+  overflow-y: scroll;
+  overflow-x: hidden;
+}
+input {
+  font-family: var(--rh-theme--font-family);
+  font-size: var(--rh-theme--font-size);
+  line-height: var(--rh-theme--line-height);
+}
+
+ul {
+  font-family: var(--rh-theme--font-family);
+  font-size: var(--rh-theme--font-size);
+  line-height: var(--rh-theme--line-height);
+  background-color: #fff;
+  border: 1px solid #ccc;
+  border-top: none;
+  margin: 0px;
+  padding: 0px;
+  list-style: none;
+  cursor: pointer;
+
+  li {
+      display: list-item;
+      cursor: pointer;
+      padding: 7px 10px;
+      margin: 2px 0px 2px 0px;
+      &.active {
+          background-color: #{rh-color( surface--lighter )};;
```

Comment:
@Homa I added some theming to this branch, though its not complete. Hope this helps :)